integration-tests/browser_integration_test.go

```diff
@@ -1,3 +1,5 @@
+// +build !js
```

Comment:
Did something change, or should this have always been here?

Reply:
This file never worked in the browser, (among other things, it uses `chromedp` and `os/exec`). But until this PR, it at least _built_ in the browser.

What changed was that I added `+build !js` to the `rpc` package to make sure that we don't include it in browser builds. As a consequence we also needed to add the tag here.

Reply:
Ah I see. That makes sense.